workspaces/arborist/lib/arborist/load-virtual.js

```diff
@@ -200,6 +200,18 @@ module.exports = cls => class VirtualLoader extends cls {
       const targetPath = resolve(this.path, meta.resolved)
       const targetLoc = relpath(this.path, targetPath)
       const target = nodes.get(targetLoc)
+
+      if (!target) {
+        const err = new Error(
+`Missing target in lock file: "${targetLoc}" is referenced by "${location}" but does not exist.
+To fix:
+1. rm package-lock.json
```

Comment:
This is actually a bit too explosive. A more graceful fix would be to just remove the reference `location` from the package lock and then run `npm i` which fixes the issue.

Reply:
It already explodes, this just lets you know why.  The fix was a better error message.

Reply:
Oops, That was supposed to be [removed](https://github.com/npm/cli/pull/8268#discussion_r2087620272)

Reply:
That comment was only about removing node modules.

Reply:
We don't ever recommend people manually edit their package-lock file. The real fix would be for `npm install` to fix the tree, but that's a much bigger lift.

Reply:
Fair enough